app/customer.py

```diff
@@ -0,0 +1,17 @@
+from app.car import Car
+
+
+class Customer:
+    def __init__(
+            self,
+            name: str,
+            products_to_buy: dict,
+            location: list[int],
+            money: int | float,
+            car: dict
+    ) -> None:
+        self.name = name
+        self.products_to_buy = products_to_buy
+        self.location = location
+        self.money = money
+        self.car = Car(car["brand"], car["fuel_consumption"])

```

Comment:
Ensure that the `car` dictionary contains the keys 'brand' and 'fuel_consumption' before initializing the `Car` object. Consider adding validation or error handling to manage cases where these keys might be missing.